src/h_vialib/secure/cookie.py

```diff
@@ -0,0 +1,122 @@
+"""Cookie helpers to create secure cookies."""
+
+from http.cookies import SimpleCookie
+
+from h_vialib.exceptions import MissingToken
+from h_vialib.secure.expiry import as_expires
+
+
+class Cookie:
+    """A simplified Cookie for security purposes."""
+
+    DATE_FORMAT = "%a, %d %b %Y %H:%M:%S %Z"
```

Comment:
```suggestion
    _DATE_FORMAT = "%a, %d %b %Y %H:%M:%S %Z"
```

I spent 72 hours trying to figure out why this was part of the public interface before finally giving up in total and utter confusion and despair

Reply:
Thus the 4 day delay on review... makes sense